### PR TITLE
Updating policy api version used in pod evictor

### PR DESCRIPTION
**Pull request by lx1036**
fix eviction APIVersion typo in test case

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/ok-to-test
These could actually probably reference the [Policy GroupVersion constant](https://github.com/kubernetes/api/blob/v0.21.0/policy/v1beta1/register.go#L29) directly, and then parse it with [GroupVersion.String()](https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime/schema#GroupVersion.String)

**Comment by damemi** [comment]
Just a nit but could you also retitle the PR to better describe that this is actually updating the Policy API version used in PodEvictor?

**Comment by lx1036** [comment]
> Just a nit but could you also retitle the PR to better describe that this is actually updating the Policy API version used in PodEvictor?

thanks, i have retitled the PR,  and updated the commit content and message.

File: pkg/descheduler/strategies/duplicates_test.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	"testing"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/policy/v1beta1"
```
Comment (seanmalloy) [inline pkg/descheduler/strategies/duplicates_test.go:24]: How about ...
```suggestion
	policyv1 "k8s.io/api/policy/v1"
```

File: pkg/descheduler/strategies/duplicates_test.go [hunk 2]
```diff
@@ -402,7 +403,7 @@ func TestRemoveDuplicatesUniformly(t *testing.T) {
 			})
 			podEvictor := evictions.NewPodEvictor(
 				fakeClient,
-				"v1",
+				v1beta1.SchemeGroupVersion.String(),
```
Comment (seanmalloy) [inline pkg/descheduler/strategies/duplicates_test.go:406]: How about ...
```suggestion
				policyv1.SchemeGroupVersion.String(),
```

**Comment by seanmalloy** [comment]
@damemi please take another look at this PR when you get a chance. Thanks!

/lgtm

**Review by seanmalloy — APPROVED** [review]

File: pkg/descheduler/strategies/duplicates_test.go (hunk 1 above)
Comment (lx1036) [reply pkg/descheduler/strategies/duplicates_test.go]: resolve it. thanks

File: pkg/descheduler/strategies/duplicates_test.go (hunk 2 above)
Comment (lx1036) [reply pkg/descheduler/strategies/duplicates_test.go]: resolve it. thanks

**Comment by lx1036** [comment]
@seanmalloy  have used "k8s.io/api/policy/v1". thanks.

**Review by damemi — COMMENTED** [review]
/approve

**Pull request merged**